basyx.aasrepository/basyx.aasrepository-core/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/AasRepositorySuite.java

```diff
@@ -287,6 +298,80 @@ public void getPaginatedSubmodelReferencesPaginated() {
 						.get());
 	}
 
+	@Test
+	public void updateThumbnail() {
```

Comment:
It is not testing the updateThumbnail behaviour

Reply:
why not?

Reply:
Here I test the setThumbnail behavior. THe shell did not has a thumbnail before, and after setThumbnail, the shell should have a thumbnail. Here I asset that the thumbnail exists.